### PR TITLE
[FIX] More web changes Newpct1

### PR DESCRIPTION
**Pull request by malkavi**
Now Newpct1 and Newpct share torrent urlrewrite.

### Motivation for changes:
Changes in newpct1

### Detailed changes:
- newpct1 and newpct share code

### CODE REVIEW

File: flexget/plugins/sites/newpct.py [hunk 1]
```diff
@@ -62,31 +62,24 @@ def parse_download_page(self, url):
...
-                if match:
-                    torrent_id = match.group(2)
         else:
             url_format = NEWPCT_TORRENT_FORMAT
 
-            torrent_id_prog = re.compile("(?:parametros\s*=\s*\n?)\s*{\s*\n(?:\s*'\w+'\s*:.*\n)+\s*'(?:torrentID|id)"
+        torrent_id_prog = re.compile("(?:parametros\s*=\s*\n?)\s*{\s*\n(?:\s*'\w+'\s*:.*\n)+\s*'(?:torrentID|id)"
                                          "'\s*:\s*'(\d+)'")
+        torrent_ids = soup.findAll(text=torrent_id_prog)
+        if len(torrent_ids):
```
Comment (liiight) [inline flexget/plugins/sites/newpct.py]: you can do just `if torrent_ids:`

File: flexget/plugins/sites/newpct.py [hunk 2]
```diff
@@ -22,8 +22,8 @@
 requests.add_domain_limiter(TimedLimiter('newpct1.com', '2 seconds'))
 requests.add_domain_limiter(TimedLimiter('newpct.com', '2 seconds'))
 
-NEWPCT_TORRENT_FORMAT = 'http://www.newpct.com/torrents/{:0>6}.torrent'
-NEWPCT1_TORRENT_FORMAT = 'http://www.newpct1.com/download/{}.torrent'
+NEWPCT_TORRENT_FORMAT   = 'http://www.newpct.com/torrents/{:0>6}.torrent'
```
Comment (liiight) [inline flexget/plugins/sites/newpct.py]: looks like there's an extra space there

File: flexget/plugins/sites/newpct.py [hunk 3]
```diff
@@ -62,31 +62,24 @@ def parse_download_page(self, url):
...
+        torrent_ids = soup.findAll(text=torrent_id_prog)
+        if len(torrent_ids):
+            match = torrent_id_prog.search(torrent_ids[0])
+            if match:
+                torrent_id = match.group(1)
+        if not torrent_id:
+            torrent_id_prog = re.compile('function openTorrent.*\n.*\{.*(\n.*)+window\.location\.href =\s*\".*\/(\d+).*\";')
             torrent_ids = soup.findAll(text=torrent_id_prog)
-            if len(torrent_ids):
+            log.debug('torrent ID not found, searching openTorrent script')
```
Comment (cvium) [inline flexget/plugins/sites/newpct.py]: This doesn't really make sense if `torrent_ids` is empty/none.

Comment (malkavi) [reply flexget/plugins/sites/newpct.py]: There are two attempts to find torrent id, first regexp with 'parametros', then regexp with opentorrent. The log is a reference to the first regexp failure. Should I remove the log?

**Comment by ciltocruz** [comment]
Sorry for noise closed topics.
I think that this is where I should write. If not, tell me where.

I didn't want to open new topics because I didn't want to overload the error issues.

**Pull request merged**